starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+	<section class="section1" >
+		
+		<div class="header">
```

Comment:
for the header you should use 

```
<header>
<h1>Apple Pies</h1>
</header>
```

it's important to use semantic tags when possible